webapp/packages/core-connections/src/module.ts

```diff
@@ -61,6 +62,7 @@ ModuleRegistry.add({
       .addSingleton(NetworkHandlerResource)
       .addSingleton(NavNodeExtensionsService)
       .addSingleton(DBDriverResource)
+      .addSingleton(DBDriverExpertSettingsResource)
```

Comment:
please also add `.addSingleton(Dependency, proxy(DBDriverExpertSettingsResource))`
This will keep the resource initialized even if it's not used